gradle/libs.versions.toml

```diff
@@ -0,0 +1,153 @@
+# Also see settings.gradle file for dynamic versionCatalogs dependencies
+
+[versions]
+antlr = "4.9.3"
+deltaStandalone = "0.6.0"
```

Comment:
to be consistent, maybe the version definitions should be in the form `delta-standalone` / `spark-hive-31` rather than `deltaStandalone` / `sparkHive31`

Reply:
done

note that I still avoided usage of version naming that contain part consisting of just numbers, I'm not sure that this idiomatic way of doing that, I checked it does work but IDE doesn't support that and jump to definitions stops working, though something like spark-hive-v31 should work

let me know what you think, Ideally it would be nice to see a version separately and also separate version into a major and minor versions parts, spark-hive-3-1 but as I said it does break current IDE support, at least in InteliJ Idea